pkg/generate/templateset/vars.go

```diff
@@ -16,24 +16,25 @@ package templateset
 // MetaVars contains template variables that most templates need access to
 // that describe the service alias, its package name, etc
 type MetaVars struct {
+	// ServiceModelName contains the exact string used to identify the AWS
+	// service API in the aws-sdk-go's models/apis/ directory. Note that some
+	// APIs this name does not match the ServiceID. e.g. The AWS Step Functions
+	// API has a ServiceID of "SFN" and a service model name of "states"...
+	ServiceModelName string
```

Comment:
If we move the service model name override to be in the `generator.yaml` file, you can populate this field value from the `cfg` field of the `Model` struct.